content/docs/0.8.1/advanced-resources/deploy/airgap.md

```diff
@@ -5,6 +5,36 @@ weight: 2
 
 CSI driver components images names and tags can be found [here.](../../../architecture/#kubernetes-csi-driver-images)
 
+## Note:
+
+Using a long registry URL may cause Longhorn installation error
+
+Longhorn manager would report errors in the log when this happened:
+```
+"instance-manager-e-xxxxxxxx" is invalid: metadata.labels: Invalid value: "<PRIVATE_REGISTRY_URL>-longhornio-longhorn-instance-manager-v1_20200301": must be no more than 63 characters
+```
+
+Longhorn instance manager pods have labels with key `longhorn.io/instance-manager-image` and value `REGISTRY_URL-USER-IMAGE_NAME-TAG`
+
+e.g  
+```
+metadata:
+  labels:
+    longhorn.io/component: instance-manager
+    longhorn.io/instance-manager-image: <PRIVATE_REGISTRY_URL>-longhornio-longhorn-instance-manager-v1_20200301
+    longhorn.io/instance-manager-type: engine
+    longhorn.io/node: <NODE_NAME>
+  name: instance-manager-e-XXXXXXXX
+```
+
+it's known Kubernetes limitaion that label value should be no more than 63 characters [here](https://kubernetes.io/docs/concepts/overview/working-with-objects/labels/#syntax-and-character-set)
+
+### Recommendation:
+it's highly recommended **not** to maniplute image tags, specially instance manager image tag e.g `v1_20200301`, since we intentionally used date so it is not assoctiated with Longhorn version.
+
+e.g `hub.example.com/lh/longhorn-instance-manager:v1_20200301`
```

Comment:
This is probably not short enough.
```
longhorn-instance-manager: hub.example.com/lh/ins-mgr:v1_20200301
longhorn-manager: hub.example.com/lh/mgr:v0.8.1
longhorn-engine: hub.example.com/lh/eng:v0.8.1
longhorn-ui: hub.examples.com/lh/ui:v0.8.1
```
etc.
Need examples for all the images we used.

Reply:
It's 57 characters long, but your images names looks more consistent.
Fixed.

Reply:
I meant:
1. We need to add examples for all the images, e.g. CSI ones are also needed
2. If the user's private registry's domain name is longer, then they might not able to use the name you suggested. So we want to keep the image name as short as possible (and still understandable).